helm/aladdin-demo/templates/server/deploy.yaml

```diff
@@ -11,7 +11,7 @@ spec:
   selector:
     matchLabels:
       app: {{ .Chart.Name }}-server
-  replicas: {{ .Values.deploy.replicas }}
+  replicas: {{ .Values.server.replicas }}
```

Comment:
our values.yaml now has a section labeled `server` and `commands` for these settings